<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qMRMLPlotViewControllerWidget</class>
 <widget class="ctkPopupWidget" name="qMRMLPlotViewControllerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plot View Controller</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <layout class="QGridLayout" name="contentLayout">
     <item row="1" column="0">
      <widget class="qMRMLNodeComboBox" name="plotChartComboBox" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>65</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Select the Plot Layout</string>
       </property>
       <property name="nodeTypes" stdset="0">
        <stringlist>
         <string>vtkMRMLPlotChartNode</string>
        </stringlist>
       </property>
       <property name="noneEnabled" stdset="0">
        <bool>true</bool>
       </property>
       <property name="addEnabled" stdset="0">
        <bool>true</bool>
       </property>
       <property name="removeEnabled" stdset="0">
        <bool>false</bool>
       </property>
       <property name="renameEnabled" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="plotTypeComboBox">
       <property name="toolTip">
        <string>Select a plot type (line, scatter, bar). This will modify the type of all the current activated Plots.  </string>
       </property>
       <item>
        <property name="text">
         <string>Line</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Line and Scatter</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Scatter</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bar</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="qMRMLCheckableNodeComboBox" name="plotDataComboBox" native="true">
       <property name="toolTip">
        <string>Add/Remove Plots from the current Plot Layout</string>
       </property>
       <property name="nodeTypes" stdset="0">
        <stringlist>
         <string>vtkMRMLPlotDataNode</string>
        </stringlist>
       </property>
       <property name="addEnabled" stdset="0">
        <bool>false</bool>
       </property>
       <property name="removeEnabled" stdset="0">
        <bool>false</bool>
       </property>
       <property name="editEnabled" stdset="0">
        <bool>false</bool>
       </property>
       <property name="renameEnabled" stdset="0">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QToolButton" name="showLegendToolButton">
       <property name="toolTip">
        <string>Show legend</string>
       </property>
       <property name="text">
        <string>Legend</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="showGridToolButton">
       <property name="toolTip">
        <string>Show grid</string>
       </property>
       <property name="text">
        <string>Grid</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="labelsLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="showTitleCheckBox">
       <property name="toolTip">
        <string>Show title</string>
       </property>
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="showXAxisLabelCheckBox">
       <property name="toolTip">
        <string>Show the X-axis label</string>
       </property>
       <property name="text">
        <string>X-axis label</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="showYAxisLabelCheckBox">
       <property name="toolTip">
        <string>Show the Y-axis label</string>
       </property>
       <property name="text">
        <string>Y-axis label</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="titleLineEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Enter a title for the chart.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="xAxisLabelLineEdit">
       <property name="toolTip">
        <string>Enter a label for the X-axis.</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="yAxisLabelLineEdit">
       <property name="toolTip">
        <string>Enter a label for the Y-axis.</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="editTitleButton">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="toolTip">
        <string>Edit the title</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="editXAxisLabelButton">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="toolTip">
        <string>Edit the X-axis label</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QToolButton" name="editYAxisLabelButton">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="toolTip">
        <string>Edit the Y-axis label</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionShow_Lines">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Lines</string>
   </property>
   <property name="toolTip">
    <string>Show lines</string>
   </property>
  </action>
  <action name="actionShow_Markers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Markers</string>
   </property>
   <property name="toolTip">
    <string>Show markers</string>
   </property>
  </action>
  <action name="actionShow_Grid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Grid</string>
   </property>
   <property name="toolTip">
    <string>Show grid</string>
   </property>
  </action>
  <action name="actionShow_Legend">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Legend</string>
   </property>
   <property name="toolTip">
    <string>Show legend</string>
   </property>
  </action>
  <action name="actionFit_to_window">
   <property name="icon">
    <iconset resource="../qMRMLWidgets.qrc">
     <normaloff>:/Icons/SlicesFitToWindow.png</normaloff>:/Icons/SlicesFitToWindow.png</iconset>
   </property>
   <property name="text">
    <string>Fit_to_window</string>
   </property>
   <property name="toolTip">
    <string>Adjust the Plot Viewer's field of view to match the extent of the Plot axes</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkPopupWidget</class>
   <extends>QFrame</extends>
   <header>ctkPopupWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLCheckableNodeComboBox</class>
   <extends>qMRMLNodeComboBox</extends>
   <header>qMRMLCheckableNodeComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qMRMLWidgets.qrc"/>
 </resources>
 <connections/>
</ui>
